akka-persistence-testkit/src/test/scala/akka/persistence/testkit/scaladsl/EventSourcedBehaviorTestKitSpec.scala

```diff
@@ -76,6 +79,17 @@ object EventSourcedBehaviorTestKitSpec {
             case IncrementWithConfirmation(replyTo) =>
               Effect.persist(Incremented(1)).thenReply(replyTo)(_ => Done)
 
+            case IncrementWithAsyncReply(replyTo) =>
+              Thread.sleep(500)
```

Comment:
Yuck, but ok here I guess.

Reply:
sorry, leftover, that is not needed, I just wanted to be sure